scripts/docker/install_additional_dependencies.sh

```diff
@@ -36,26 +35,26 @@ function install_additional_dependencies() {
         echo
         echo Installing additional dependencies while upgrading to newer dependencies
         echo
-        pip install ${AIRFLOW_INSTALL_USER_FLAG} --upgrade --upgrade-strategy eager \
+        pip install --upgrade --upgrade-strategy eager \
             ${ADDITIONAL_PYTHON_DEPS} ${EAGER_UPGRADE_ADDITIONAL_REQUIREMENTS}
         # make sure correct PIP version is used
-        pip install ${AIRFLOW_INSTALL_USER_FLAG} --upgrade "pip==${AIRFLOW_PIP_VERSION}"
```

Comment:
Latest pip uses `--user` by default?

Reply:
Nope we have
```
    PIP_USER="true"
```

Reply:
Apparently looks like I even added comment about it :)
```
# By default PIP will install everything in ~/.local
ARG PIP_USER="true"
```